build-all

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+#
+# Script to build all (or most) libyui subprojects in the correct order.
+#
+# Author:  Stefan Hundhammer <shundhammer@suse.com>
+# License: GPL V2
+
+script_name=$(basename $0)
+dry_run=0
+verbose=0
+
+# subprojects that will or will not be built
+with_qt=1
+with_ncurses=1
+with_pkg=1
+with_graph=1
+with_rest_api=0
+with_bindings=0
+
+
+# Show usage information and exit.
+function usage()
+{
+    cat <<EOF
+
+Usage:
+
+  $script_name [Options] [configure|build|install|clean|doc]
+
+Build all (or most) libyui subprojects in the correct order.
+Without options, this builds the 'typical' set of subprojects.
+To see which ones that are, use
+
+  ./$script_name -n
+
+If invoked without any target argument, this script just executes 'make' in
+each subproject that already has a build/ subdirectory. For subprojects
+without such a build/ subdirectory, it behaves as if invoked with 'build'.
+
+Notice that for installing, you need to call this with root privileges, i.e.
+
+  sudo ./$script_name install
+
+Options:
+
+  Component selection:
+
+  -a, --all         Build with all subprojects
+  -m, --min         Minimalistic: libyui, -qt, -ncurses, -bindings
```

Comment:
libyui-bindings being included in `--min` but excluded by default is inconsistent

Reply:
Any suggestions for a better name for this? It's mostly intended for non-YaST / non-SUSE use cases.

And it _should_ be a free single letter, and we should keep `-n` for `--dry-run` like in `make` or `rsync` and `-k` (which I just added) like in `make`.

Reply:
How about `-s` / `--small` ?

Reply:
Now using `-s` / `--small` instead.